### DISCUSSION TITLE
An error when runned applications/ChatGPT/examples/train_sft.py

### DISCUSSION THREAD

**Discussion by guijuzhejiang** [hpcaitech/ColossalAI#3224] [Community | Q&A]
Excuting：
python train_sft.py --pretrain facebook/opt-350m --model opt --strategy naive --log_interval 10
Error message:
RuntimeError: Default process group has not been initialized, please make sure to call init_process_group.
It seems that it cannot run on a single GPU. May I ask the parameter settings when running on a single GPU. 
Also, is train_sft.py the training code of stage1 of chatgpt?

**Comment by binmakeswell** [comment]
Hi @guijuzhejiang Thanks for your feedback. It's still under development, maybe you can try it again next week. Thanks.
